lxd/container_lxc.go

```diff
@@ -5061,14 +5052,9 @@ func (c *containerLXC) Update(args db.ContainerArgs, userRequested bool) error {
 		}
 
 		updateDiskLimit := false
-		for k, m := range updateDevices {
+		for _, m := range updateDevices {
 			if m["type"] == "disk" {
 				updateDiskLimit = true
-			} else if m["type"] == "proxy" {
-				err = c.updateProxyDevice(k, m)
```

Comment:
@stgraber looking at the old fields list that were considered to be "live updatable", none of the proxy related fields were in that list, so I don't believe this part of the code was reachable.

Reply:
Interesting, in both cases we'd just kill the proxy and respawn it, so unlikely to be a bug.

Reply:
@stgraber yes I dont think it causes a problem, but it will be removed as part of this PR.